src/Templates/SearchResultsProductTemplate.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\Templates;
+
+/**
+ * ProductSearchTemplate class.
+ *
+ * @internal
+ */
+class SearchResultsProductTemplate {
```

Comment:
Just curious, any reason why this is named `SearchResultsProductTemplate`? `ProductSearchResultsTemplate` would seem more natural to me.

Reply:
Not really. I'm not sure about the `search-results-product.html` filename too. I just used this for being consistent, but thinking about it, my assumption doesn't make a lot of sense 😅

![image](https://user-images.githubusercontent.com/4463174/163005159-54732af6-bfbe-49d9-8e39-a0064b2f796f.png)